examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -114,9 +117,27 @@ void DeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, intptr_
         ESP_LOGI(TAG, "CHIPoBLE disconnected");
         break;
 
-    case DeviceEventType::kCommissioningComplete:
+    case DeviceEventType::kCommissioningComplete: {
         ESP_LOGI(TAG, "Commissioning complete");
-        break;
+#if CONFIG_BT_NIMBLE_ENABLED && CONFIG_DEINIT_BLE_ON_COMMISSIONING_COMPLETE
+        int ret = nimble_port_stop();
+        if (ret == 0)
+        {
+            nimble_port_deinit();
+            esp_err_t err = esp_nimble_hci_and_controller_deinit();
+            err           = esp_bt_mem_release(ESP_BT_MODE_BLE);
+            if (err == ESP_OK)
+            {
+                ESP_LOGI(TAG, "BLE deinit successful");
```

Comment:
BLE deinit successful and memory reclaimed?

Reply:
done